PROTOCOL.md

```diff
@@ -386,8 +386,8 @@ The histogram bins correspond to the following ranges:
 - Bin 5: [10000, 99999] (files with 10,000-99,999 deleted records)
 - Bin 6: [100000, 999999] (files with 100,000-999,999 deleted records)
 - Bin 7: [1000000, 9999999] (files with 1,000,000-9,999,999 deleted records)
-- Bin 8: [10000000, Int.MaxValue - 1] (files with 10,000,000 to Int.MaxValue-1 deleted records)
-- Bin 9: [Int.MaxValue, Long.MaxValue] (files with Int.MaxValue or more deleted records)
+- Bin 8: [10000000, 2147483646] (files with 10,000,000 to 2147483646 (i.e. Int.MaxValue-1 in Java) deleted records)
+- Bin 9: [2147483647, 9,223,372,036,854,775,807] (files with 2147483647 or more deleted records)
```

Comment:
While [2147483647, 9,223,372,036,854,775,807] is effectively the same as >= 2147483647, I feel that it would be better to at least have some bound here.

Reply:
I thought about this again and you are right, it doesn't make sense to specify an upper bound for the last bucket